Show "Tuning Performance" guide for all builds (#4170)
theforeman/foreman-documentation: guides/common/modules/con_smart-proxy-configuration-tuning.adoc

= {SmartProxy} configuration tuning

{SmartProxies} are meant to offload part of {Project} load and provide access to different networks related to distributing content to clients but they can also be used to execute remote execution jobs.

> What they cannot help with is anything which extensively uses {Project} API,
> ifdef::katello,orcharhino,satellite[]
> for example, host registration or package profile update.
> endif::[]
> ifdef::foreman-el,foreman-deb[]
> for example, host registration.
> endif::[]

Reviewer: ```suggestion
What they cannot help with is anything which extensively uses {Project} API,
ifdef::katello,orcharhino,satellite[]
such as host registration or package profile update.
endif::[]
ifdef::foreman-el,foreman-deb[]
such as host registration.
endif::[]
```
To reduce the number of commas because they made the result a bit difficult to parse.
